tests/sktime/test_sktime_check_estimator.py

```diff
@@ -1,7 +1,7 @@
 """Test the sktime contract for Prophet and HierarchicalProphet."""
 
 import pytest
-from sktime.utils.estimator_checks import check_estimator
+from sktime.utils.estimator_checks import check_estimator, parametrize_with_checks
 
 from prophetverse.sktime import HierarchicalProphet, Prophetverse
```

Comment:
if you import all the other models, parametrize them as `PROPHET_MODELS` and add a `get_test_params` to each would cause the tests to run without the existing issue.